nix/arweave.nix

```diff
@@ -451,16 +419,10 @@ let
         mkdir -p apps/arweave/lib/RandomX
         cp -rf ${randomx}/* apps/arweave/lib/RandomX
         cp -rf ${jiffy}/lib/erlang/lib/* apps/jiffy
-        cp -rf ${graphql}/lib/erlang/lib/* apps/graphql
       '';
 
       postPatch = ''
-        sed -i -e 's|-arch x86_64|-arch ${pkgs.stdenv.targetPlatform.linuxArch}|g' \
-          apps/arweave/c_src/Makefile \
```

Comment:
do we not need `apps/arweave/c_src/Makefile` any more?

Reply:
good catch, we do, I should have just removed apps/ar_sqlite3/c_src/Makefile

Reply:
fixed